test/e2e/framework/meta.go

```diff
@@ -7,12 +7,14 @@ import (
 )
 
 const (
-	SerialLabelName = "Serial"
+	SerialLabelName            = "Serial"
+	RequiresClusterIPLabelName = "RequiresClusterIP"
```

Comment:
`ClusterIPOnly` would read nicely in the logs

Reply:
Wouldn't it be misleading in regard to the service type though? E.g. service of type LoadBalancer (or any other extension of ClusterIP) would also handle these tests

Reply:
it's a label for a particular test that can only run with scyllaclusters using cluster ip exposure type. (`Only` is often used as suffix for test that are skipped when the config/platform doesn't support it)

Reply:
or were you pointing to the need for type prefix like `SCClusterIPOnly`?

Reply:
I guess in can be ambiguous in a different place. `RequiresClusterIP` is fine.

Reply:
> it's a label for a particular test that can only run with scyllaclusters using cluster ip exposure type.

for the NodeServiceType that would also be misleading, no? As the `NodeServiceTypeLoadBalancer` would also handle it